frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboardView-filters.stories.tsx

```diff
@@ -119,7 +119,7 @@ function FasterExplicitSizeUpdateDecorator(Story: Story) {
  * This value works together with FasterExplicitSizeUpdateDecorator which
  * make sure we finish resizing any ExplicitSize components the fastest.
  */
-const TIME_UNTIL_ALL_ELEMENTS_STOP_RESIZING = 1000;
+const TIME_UNTIL_ALL_ELEMENTS_STOP_RESIZING = 1500;
```

Comment:
afaik `ExplicitSize` has different strategies of delaying re-renderings (debounce, throttle etc) and for cypress it's completely disabled. should we disabled any delay for storybook/loki as well?

please check https://github.com/metabase/metabase/blob/master/frontend/src/metabase/components/ExplicitSize/ExplicitSize.tsx#L122-L129

Reply:
It's already disabled here
https://github.com/metabase/metabase/blob/23725b4e9dcde164294b34966d3246ea3a909d08/frontend/src/metabase/components/ExplicitSize/ExplicitSize.tsx#L77-L79

Reply:
Even with that disabled we still need to wait for this on the CI from my testing unfortunately.

Reply:
okay, thanks for explanation, but anyway, having a hardcoded value is not a good solution here. Is it possible to have some promise that will be resolved only when resizing is finished?

Reply:
If we could do that then it'd be great. I couldn't figure out how to get that info/create that kind of promise though.